internal/hardware/hardware.go

```diff
@@ -1,29 +1,29 @@
 package hardware
 
 import (
-	"encoding/json"
 	"github.com/jakub-dzon/k4e-operator/models"
 	"github.com/openshift/assisted-installer-agent/src/inventory"
+	"github.com/openshift/assisted-installer-agent/src/util"
 )
 
 type Hardware struct {
 }
 
 func (s *Hardware) GetHardwareInformation() (*models.HardwareInfo, error) {
-	inv := inventory.ReadInventory(&inventory.Options{
-		GhwChrootRoot: "/",
-	})
+	d := util.NewDependencies("/")
+	cpu := inventory.GetCPU(d)
+	hostname := inventory.GetHostname(d)
+	systemVendor := inventory.GetVendor(d)
```

Comment:
as we learned, we'll have to collect and report IP addresses as well.

Reply:
fixed